### PR TITLE
RE: #33 - add sponsor-type, remove is-internal

### PR DESCRIPTION
**Pull request by jvasquez75**

### CODE REVIEW

**Comment by jvasquez75** [comment]
@gdbrown - xmod updated

File: view/cms/templates/wb-sponsors-edit.xmod [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <fieldset>
     <title>Integration Code</title>
     <group>
-      <checkbox id="#is-internal" width="full"/>
+      <checkbox id="#sponsor-type" width="full" rows="5"/>
```
Comment (gdbrown) [inline view/cms/templates/wb-sponsors-edit.xmod]: @jvasquez75 rows only applies to textareas

Comment (gdbrown) [reply view/cms/templates/wb-sponsors-edit.xmod]: @jvasquez75 also, it should be a textbox

Comment (jvasquez75) [reply view/cms/templates/wb-sponsors-edit.xmod]: actually i wanted to be a dropdown!

**Pull request merged**